webserver/webserver/src/main/java/io/helidon/webserver/netty/NettyWebServer.java

```diff
@@ -55,7 +55,7 @@
 /**
  * The Netty based WebServer implementation.
  */
-class NettyWebServer implements WebServer {
+public class NettyWebServer implements WebServer {
```

Comment:
This class should not be public. The NettyWebServer class should be in the same package as the rest (e.g. `io.helidon.webserver`).
According to the development guidelines we should use flat package structure.
Adding a public class makes this part of our API

Reply:
Forgot to say "will fix" ;)